fs/xfstests.py.data/xfstests.yaml

```diff
@@ -1,4 +1,16 @@
 setup:
-    runargs : !mux
-        skip_dangerous: True
-        test_range: '2,12,34-87,91,99-112'
+    skip_dangerous: True
+    file_system: 'ext4'
+
+    # Uncomment and edit test_range for running specific tests
+    # test_range: '73,217-415'
+
+    # Run with either loop_type (or) disk_type
+    loop_type: !mux
+        type: 'loop'
+        base_disk: /dev/sdj2
```

Comment:
@narasimhan-v @PraveenPenguin It is okay to change. I have added a separate type for direct disks, currently in `TODO`